src/app/shared/components/card/card.component.js

```diff
@@ -50,12 +55,16 @@ class Card extends Component {
     }
 
     handleClick() {
-        this.props.onCardClick(this.props.card);
+        this.props.moveCardDriver2(this.props.card);
+        // this.props.onCardClick(this.props.card);
         // this.setState(prevState => ({
         //     isOnBackSide: !this.state.isOnBackSide
         // }));
     };
 
+    flipCard() {
+        this.setState({isOnBackSide: !this.state.isOnBackSide})
+    }
```

Comment:
I suggest we don't deel with flip card for now. Let's finish developing the game with visible cards, it will make our lives easier

Reply:
done